<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>541</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>199</width>
       <height>539</height>
      </rect>
     </property>
     <widget class="QListWidget" name="listLists">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTableWidget" name="listItemTable">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>1</y>
      <width>631</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_item">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>520</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter un Item</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuListe">
    <property name="title">
     <string>Liste</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDelete"/>
   </widget>
   <addaction name="menuListe"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>listLists</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>showList(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>917</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showList(QListWidgetItem*)</slot>
 </slots>
</ui>
